woocommerce-payments.php

```diff
@@ -323,3 +323,17 @@ function wcpay_tasks_init() {
 }
 
 add_action( 'plugins_loaded', 'wcpay_tasks_init' );
+
+/**
+ * As the class is defined in later versions of WC, Psalm infers error.
+ *
+ * @psalm-suppress UndefinedClass
+ */
+add_action(
+	'before_woocommerce_init',
+	function() {
+		if ( class_exists( '\Automattic\WooCommerce\Utilities\FeaturesUtil' ) ) {
+			\Automattic\WooCommerce\Utilities\FeaturesUtil::declare_compatibility( 'custom_order_tables', __FILE__, false );
```

Comment:
We can hardcode the plugin name to `woocommerce-payments/woocommerce-payments.php` based on this discussion for atomic sites: p1665562457707829/1665514564.244559-slack-C9QDV4TL4

Reply:
Done